test/e2e/conftest.py

```diff
@@ -130,7 +134,7 @@ def create_stage_request(self, body):
         return create_stage_response
 
     def delete_stage_request(self, stage):
-        delete_stage_response = requests.delete(self.stack_resources["WorkflowApiEndpoint"]+'/workflow/stage/'+stage, verify=False, auth=self.auth)
+        delete_stage_response = requests.delete(self.stack_resources["WorkflowApiEndpoint"]+'/workflow/stage/'+stage+'?force=true', verify=False, auth=self.auth)
```

Comment:
This force=true option is needed so that pytests don't fail when they try to delete stages which are dependencies for other stages. With this force option enabled, pytests can cleanup all stages at once, and not have to worry about waiting for dependent stages to delete. All their stages are test stages anyway.